dts/arm/atmel/samd5x.dtsi

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright (c) 2019 ML!PA Consulting GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/i2c/i2c.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m4";
+			reg = <0>;
+		};
+	};
+
+	sram0: memory@20000000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x20000000 0x40000>;
+	};
+
+	soc {
+		nvmctrl: nvmctrl@41004000  {
+			compatible = "atmel,sam0-nvmctrl";
+			label = "FLASH_CTRL";
+			reg = <0x41004000 0x22>;
+			interrupts = <29 0>, <30 0>;
+
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			flash0: flash@0 {
+				compatible = "soc-nv-flash";
+				label = "FLASH_0";
+				reg = <0 0x100000>;
+				write-block-size = <64>;
+			};
+		};
+
+		dma: dmac@4100A000 {
+			compatible = "atmel,sam0-dmac";
+			reg = <0x4100A000 0x50>;
+			interrupts = <31 0>, <32 0>, <33 0>, <34 0>;
```

Comment:
I think you're missing IRQ 35 here (all DMA channels from 4..31).

Reply:
good catch!